packages/dart_firebase_admin/test/auth/token_verifier_test.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:dart_firebase_admin/src/auth.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('DecodedIdToken', () {
+    test('.fromMap', () async {
+      final idToken = DecodedIdToken.fromMap(
+        {
+          'aud': 'mock-aud',
+          'auth_time': 1,
+          'email': 'mock-email',
+          'email_verified': true,
+          'exp': 1,
+          'firebase': {
+            'identities': {
+              'email': 'mock-email',
+            },
+            'sign_in_provider': 'mock-sign-in-provider',
+            'sign_in_second_factor': 'mock-sign-in-second-factor',
+            'second_factor_identifier': 'mock-second-factor-identifier',
+            'tenant': 'mock-tenant',
+          },
+          'iat': 1,
+          'iss': 'mock-iss',
+          'phone_number': 'mock-phone-number',
+          'picture': 'mock-picture',
+          'sub': 'mock-sub',
+          'uid': 'mock-sub',
```

Comment:
The doc says that `uid` isn't actually present in the token, and this field is just a convenience that takes its value from `sub`. How about removing this line then?

Reply:
Good catch